src/infrastructure/database/migrations/versions/2024_04_01_17_46-add_new_field_tags_to_subject.py

```diff
@@ -0,0 +1,35 @@
+"""Add new field tags to subject
+
+Revision ID: 809b07a71159
+Revises: 01115b529336
+Create Date: 2024-04-01 17:46:27.821161
+
+"""
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "809b07a71159"
+down_revision = "01115b529336"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "subjects",
+        sa.Column(
+            "tag",
+            sqlalchemy_utils.types.encrypted.encrypted_type.StringEncryptedType(),
```

Comment:
Do we really need to make the field encryptable? Is the value supposed to contain some PII?

Reply:
Good questions, since value will be Guardian, child, teacher, ... it can exposes subject, but I will ask other about it

Reply:
remove encryption, good catch!